styles/electrified.mapcss

```diff
@@ -369,12 +375,14 @@ node|z17-[railway=signal]["railway:signal:direction"]["railway:signal:electricit
 	allow-overlap: true;
 }
 
-/***************************/
-/* DE pantograph down El 4 */
-/* AT Stromabnehmer tief   */
-/***************************/
+/*******************************/
+/* DE pantograph down El 4     */
+/* AT Stromabnehmer tief       */
+/* NL stroomafnemers neer 309a */
+/*******************************/
 node|z17-[railway=signal]["railway:signal:direction"]["railway:signal:electricity"="DE-ESO:el4"]["railway:signal:electricity:type"="pantograph_down"]["railway:signal:electricity:form"="sign"],
 node|z17-[railway=signal]["railway:signal:direction"]["railway:signal:electricity"="AT-V2:stromabnehmer_tief"]["railway:signal:electricity:type"="pantograph_down"]["railway:signal:electricity:form"=sign]
```

Comment:
comma

Reply:
same mistake twice, sorry

It's fixed now